app/services/dunning_campaigns/update_service.rb

```diff
@@ -94,6 +83,25 @@ def reset_customers_if_no_threshold_match
         end
     end
 
+    def customers_to_reset
+      @customers_to_reset ||= customers_applied_campaign.or(customers_fallback_campaign)
+    end
+
+    def customers_applied_campaign
+      organization
+        .customers
+        .with_dunning_campaign_not_completed
+        .where(applied_dunning_campaign: dunning_campaign)
+    end
+
+    def customers_fallback_campaign
+      organization
+        .customers
+        .falling_back_to_default_dunning_campaign
+        .with_dunning_campaign_not_completed
+        .where(dunning_campaign.applied_to_organization ? nil : "1 = 0")
```

Comment:
still not sure to understand 😅

Reply:
hahaha, sorry, that one came back from the previous change, I'm going to drop it one more time and for the very last time.